gtfs/Governance/guiding-principles.md

```diff
@@ -0,0 +1,18 @@
+# Guiding principles
+
+In order to preserve the original vision of GTFS, a number of guiding principles have been established to take into consideration when changing the specification:
+
+**Feeds should be easy to create and edit**  
+We chose CSV as the basis for the specification because it's easy to view and edit using spreadsheet programs and text editors, which is helpful for smaller agencies. It's also straightforward to generate from most programming languages and databases, which is good for publishers of larger feeds.
+
+**Feeds should be easy to parse**  
+Feed readers should be able to extract the information they're looking for with as little work as possible. Changes and additions to the feed should be as broadly useful as possible, to minimize the number of code paths that readers of the feed need to implement. (However, making creation easier should be given precedence, since there will ultimately be more feed publishers than feed readers.)
+
+**The spec is about passenger information**  
+GTFS is primarily concerned with passenger information. That is, the spec should include information that can help power tools for riders, first and foremost. There is potentially a large amount of operations-oriented information that transit agencies might want to transmit internally between systems. GTFS is not intended for that purpose and there are potentially other operations-oriented data-standards that may be more appropriate.
```

Comment:
I suggest removing both instances of the word "potentially". But completely agree with the underlying idea here.